client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/rpc/ScheduleTaskInvocationDto.java

```diff
@@ -0,0 +1,54 @@
+package org.evomaster.client.java.controller.api.dto.problem.rpc;
+
+import java.util.List;
+
+public class ScheduleTaskInvocationDto {
```

Comment:
this is under `rpc` package. is it something specific only for RPC? or technically could be used for REST and GraphQL as well? if so, might be best to have in a different package

Reply:
or if it is something temporary and adhoc for Meituan, which will need to be refactored in the future, can leave some TODO comments about it